### DISCUSSION TITLE
Slice Pasef

### DISCUSSION THREAD

**Discussion by Mickey9292** [vdemichev/DiaNN#831] [General]
Hi,

We are using the SlicePasef method with one frame. Why do we have to use the command line « —tims-scan »  ? I thought this command was used for the demultiplexing step but it also improves the results with the 1F method and there is no overlap in this method. 
Tanks  for your help!

**Comment by vdemichev** [comment]
Slice-PASEF is processed with a different module in DIA-NN which is activated with --tims-scan, i.e. this is a command specifically for Slice-PASEF.

**Comment by Marine0258** [comment]
Hello Mr Demichev, 

First of all, thank you for creating a tool as powerful as DIA-NN and open to all. I'm currently working with a Bruker timsTOF Pro and I've tested the slice-PASEF method. Thanks again for this superb method. 

I downloaded DIA-NN version 1.8.2 BETA 11 to process my slice-PASEF data, but I can't find the -tims-scan? function. 

Where can I find it?

Thanks in advance!

**Reply by vdemichev** [reply]
Please put it in 'Additional options'.
Also, I recommend using beta 27 instead.

Best,
Vadim
